assets/templates/partials/list.tmpl

```diff
@@ -0,0 +1,57 @@
+{{ $lang := .Language }}
+{{ $currentPage := .Data.Pagination.CurrentPage }}
+{{ $itemsPerPage := .Data.Pagination.Limit }}
+{{ $totalSearchPosition := multiply (subtract $currentPage 1) $itemsPerPage }}
+{{ $response := .Data.Response }}
+{{ $datasetFinder := .Data.PopulationTypeFilter }}
+
+<div id="results">
+    <div id="results-loading" class="ons-panel ons-panel--info ons-panel--no-title hide">
+        <span class="ons-u-vh">{{ localise "ImportantInformation" $lang 1 }}</span>
+        <div class="ons-panel__body">
+        <p data-error-message="{{ localise "ResultsFailedToLoad" $lang 1 }}">
+            {{ localise "LoadingResults" $lang 1 }}
+        </p>
+        </div>
+    </div>
+    <div id="results-zero" class="hide">
+        <h3>{{ localise "NoResultsHeader" $lang 1 }}</h3>
```

Comment:
Given we've got no filters here - should we be prompting to clear the filters?

Reply:
Same comment on the re-using the no-results body text from the toml file

Reply:
👍